doc/modules/mod_keystore.md

```diff
@@ -20,18 +20,19 @@ for each virtual XMPP domain to avoid sharing keys between domains!**
 
 ### Options
 
-* `ram_key_size`: size to use when generating RAM-only keys (designated
-    by type `ram`)
-* `keys`: list of _specifiers_ of keys which will be provided by the
-    module at runtime
+#### `modules.mod_keystore.ram_key_size`
+* **Syntax:** non-negative integer
+* **Default:** `1000`
```

Comment:
What should be default value here?

Reply:
2048?
https://github.com/esl/MongooseIM/blob/84efd3addbe3f48ee533f27982ec6a23af70ccad/src/mod_keystore.erl#L28